web/test/pages/profile.test.tsx

```diff
@@ -1727,6 +1727,29 @@ describe("profile", () => {
     });
   });
 
+  describe("Information Section", () => {
+    describe("business structure field", () => {
+      it("displays Limited Liability Company (LLC) when legal structure id is limited-liability-company", () => {
```

Comment:
I think it'd be helpful to rename this test to be more descriptive of what we care about:
- LLC is just one example of a case, the test isn't specific to LLC. we care about displaying the _name_ of the legal structure instead of the _ID_
- This test is specifically about when the field is locked, which happens after the user has completed formation. The test should specify then the scenario under test here is regarding after formation has completed. I think if the test name had that, it would have clarified soma's question without him having to ask. The test name should make that obvious so that when an engineer comes across this test in 6 months, they ALSO understand why you had to set getFilingResponse